sys/syz-sysgen/sysgen.go

```diff
@@ -313,11 +313,13 @@ func newSyscallData(target *targets.Target, sc *prog.Syscall, attrs []uint64) Sy
 	if !patchCallName {
 		callName = sc.CallName
 	}
+	needCall := (!target.SyscallNumbers || strings.HasPrefix(sc.CallName, "syz_") ||
```

Comment:
I think we now need to set target.SyscallNumbers=false for openbsd.
SyscallNumbers is used in few other places and I think all of them need to be affected.

Reply:
I started working on this in https://github.com/blackgnezdo/syzkaller/commits/libc-stubs
Unless somebody beats me to it, I intend to finish it "soon".